lib/gpfpiso.gi

```diff
@@ -1851,8 +2053,120 @@ local isob,isos,iso,gens,u,a,rels,l,i,j,bgens,cb,cs,b,f,k,w,monoid,
   return iso;
 end);
 
+# creator function for data -- for documentation
```

Comment:
Does this refer to `grp/simplerew.grp`, i.e. did this code generate `grp/simplerew.grp`? If so it'd be really helpful if
the comment said that. Even better if `grp/simplerew.grp` also said that the code used to generate it can be found here / is in function ConfluentRewritingData :-)

Reply:
Added.